docs/content/references/cli/client.mdx

```diff
@@ -28,7 +29,7 @@ Commands:
   faucet                      Request gas coin from faucet. By default, it will use the active address and the active network
   gas                         Obtain all gas objects owned by the address. An address' alias can be used instead of the address
   merge-coin                  Merge two coin objects into one coin
-  new-address                 Generate new address and keypair with keypair scheme flag {ed25519 | secp256k1 | secp256r1} with optional derivation path, default to m/44'/784'/0'/0'/0' for ed25519 or m/54'/784'/0'/0/0
+  new-address                 Generate new address and key pair with key pair scheme flag {ed25519 | secp256k1 | secp256r1} with optional derivation path, default to m/44'/784'/0'/0'/0' for ed25519 or m/54'/784'/0'/0/0
```

Comment:
couple of things...this is console output so should probably update that as well (sui/crates/sui/src/client_commands.rs). Other thing, fyi, would like to eventually automate this content so it stays up2date.